lessons/Data-Science/AI/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,335 @@
+
+Debrief the activity by posing the questions below. Note that the provided answers correspond with the sample diagram (on the right). Adjust responses to match the diagram that your class produces!
+}
+
+@QandA{
+
+@Q{How many questions did we ask in order to determine the correct object?}
+@A{4}
+
+@Q{Which items from our original list (spoon, knife, cup, plate, fork, and mug) did our very first question eliminate from the list of possibilities?}
+@A{There are four items on the list with handles (spoon, knife, fork, and mug). That means our first question--"does it have a handle?" eliminated two items, cup and plate.}
+
+@Q{Can anyone think of a different question that would have eliminated more items right off the bat?}
+@A{Responses will vary. "Is it a utensil?" would have split the list in half, given that three items (spoon, knife, and fork) are utensils.}
+
+@Q{How did we decide which questions to ask?}
+@A{We had to keep track of which items were eliminated and which items remained in order to pose useful questions.}
+}
+
+Let's play _another_ round of the game with a new item. 
+
+@QandA{
+@Q{How many questions did we ask in order to determine the correct object this time?}
+@Q{How did we decide which questions to ask?}
+@Q{Which items from our original list (spoon, knife, cup, plate, fork, and mug) did our very first question eliminate from the list of possibilities?}
+@Q{How are the diagrams we drew similar and how are they different?}
+}
+
+Let's imagine that our first round had started with the question, "Is it a utensil?" and had led us to "knife".  After the first round, our tree might have looked the diagram on the left (below). If the second round started with the same question, we could have just added to the original diagram... and we might have ended up with something like what you see on the right.
+
+[cols="^.2a,<.^1a,3a", grid="none", frame="none", stripes="none"]
+|===
+
+| @hspace{8em}**Round 1**
+
+|
+
+| @hspace{8em}**Round 2**
+
+| @image{images/tree1.png, 200}
+
+| @image{images/arrow.png, 50}
+
+| @image{images/tree2.png, 370}
+
+|===
+
+
+Notice that the topmost question—"is it a utensil?"—now splits left ("yes, it is a utensil") *and* right ("no, it is not a utensil").
+
+Our diagram begins with two unique pathways from the top of the tree to two unique items.
+
+Put another way: __There are two branches that split from the tree's root node. Each branch leads to a decision node, which eventually leads to a leaf node.__
+
+@lesson-instruction{
+Let's identify the root node, branches, decision nodes, and leaf nodes on our tree so far.
+}
+
+@teacher{Discuss your in-progress tree to help students locate the different parts. The parts are intuitively named, but formal definitions are below if needed.}
+
+@right{@image{images/terminology-tree.png, 300}}
+
+- The root node is the very top node that represents the entire population or sample.
```

Comment:
Are you sure that the root node represents the entire population? It looks like a question in the other diagrams.